### PR TITLE
WIP: Create 2023-03-10-NIH-Renewal

### PR DESCRIPTION
**Pull request by mrshirts**
Posting NIH grant renewal

### CODE REVIEW

File: content/community/news/science-updates/2023-03-10-NIH-Renewal.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+We've been very thankful to recieve significant funding from the NIH Institute of General Medical Sciences for OpenFF over the last three years 
+from the Focused Technology Research and Development program. We recently submitted a proposal to renew our funding for four more years, once the 
+current grant ends next year. In the interest, of open science, we are [posting the scientific parts of the proposal](../../../science/research/2023_03_06_Open_Force_Field_Initiative_NIH_R01.pdf).
+
+We feel that overall, as part of an open science project, it's better to be open about our plans rather than keep them secret. 
+Public disclosure gives us freedom to operate in the event someone else comes into the space and to some extent protects us from commercial competitors, 
+as the ideas are publicly disclosed. It's true that someone else could scoops us, on a particlar project; but if they make their work open, then we 
+can move on to more of the other things that need to be done, and if it's closed; we'll it probably won't be that useful to people down the road anyway.
```
Comment (j-wags) [inline content/community/news/science-updates/2023-03-10-NIH-Renewal.md:22]:
```suggestion
can move on to more of the other things that need to be done, and if it's closed; well it probably won't be that useful to people down the road anyway. 
```

**Review by j-wags — CHANGES_REQUESTED** [review]
Looks like the PDF didn't end up getting added in any of the commits - Could you try adding, committing, and pushing it again?

File: content/community/news/science-updates/2023-03-10-NIH-Renewal.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+slug: 'NIH-renewal-2023-03-10'
+weight: 10
+markup: markdown
+thumb: "OpenFF-Initiative.png"
+author: "Michael Shirts"
+---
+
+We've been very thankful to recieve significant funding from the NIH Institute of General Medical Sciences for OpenFF over the last three years 
+from the Focused Technology Research and Development program. We recently submitted a proposal to renew our funding for four more years, once the 
+current grant ends next year. In the interest, of open science, we are [posting the scientific parts of the proposal](../../../science/research/2023_03_06_Open_Force_Field_Initiative_NIH_R01.pdf).
```
Comment (jchodera) [inline content/community/news/science-updates/2023-03-10-NIH-Renewal.md:17]: There should be no commas between "In the interest, of open science, we are..."

File: content/community/news/science-updates/2023-03-10-NIH-Renewal.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+draft: true
+description: 'NIH Renewal'
+slug: 'NIH-renewal-2023-03-10'
+weight: 10
+markup: markdown
+thumb: "OpenFF-Initiative.png"
+author: "Michael Shirts"
+---
+
+We've been very thankful to recieve significant funding from the NIH Institute of General Medical Sciences for OpenFF over the last three years
```
Comment (jchodera) [inline content/community/news/science-updates/2023-03-10-NIH-Renewal.md:15]: Change
```
We've been very thankful to recieve significant funding from the NIH Institute of General Medical Sciences for OpenFF over the last three years from the Focused Technology Research and Development program.
```
to
```
Over the past three years, Open Force Field Initiative activities focused on the development of general biomolecular force fields have been supported in part by [NIH grant R01 GM132386](https://reporter.nih.gov/search/kRW6qf5IVU2C_RfGNJmaIg/project-details/10166314) through the [NIGMS Focused Technology Research & Development R01 program](https://grants.nih.gov/grants/guide/pa-files/par-19-253.html).
```

File: content/community/news/science-updates/2023-03-10-NIH-Renewal.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+markup: markdown
+thumb: "OpenFF-Initiative.png"
+author: "Michael Shirts"
+---
+
+We've been very thankful to recieve significant funding from the NIH Institute of General Medical Sciences for OpenFF over the last three years 
+from the Focused Technology Research and Development program. We recently submitted a proposal to renew our funding for four more years, once the 
+current grant ends next year. In the interest, of open science, we are [posting the scientific parts of the proposal](../../../science/research/2023_03_06_Open_Force_Field_Initiative_NIH_R01.pdf).
+
+We feel that overall, as part of an open science project, it's better to be open about our plans rather than keep them secret.
```
Comment (jchodera) [inline content/community/news/science-updates/2023-03-10-NIH-Renewal.md:19]: We don't need to explain any of this. We just need to lead by example.

I'd cut this whole paragraph.

**Review by jchodera — CHANGES_REQUESTED** [review]
I think we should rewrite this.

**Comment by jchodera** [comment]
Superceded by #448

**Pull request closed without merging**